babel/localedata.py

```diff
@@ -24,15 +25,22 @@
 _dirname = os.path.join(os.path.dirname(__file__), 'locale-data')
 
 
+def normalize_locale(name):
+    """Return a normalized locale ID or `None` if the ID is not recognized."""
```

Comment:
Sure. Done in https://github.com/kdeldycke/babel/commit/a5cb8317626b71d532f78c8d86c6b0e790ad4b7f .